jsk_denso_robot/sample_pack/canon_camera/src/bcapclient.py

```diff
@@ -0,0 +1,840 @@
+﻿'''

```

Comment:
did you copy file from https://github.com/DENSORobot/orin_bcap/tree/master/Python/bCAPClient ?
note denso only provide C code for ros package , https://github.com/DENSORobot/denso_robot_ros/tree/master/bcap_core